chart/compass/charts/external-services-mock/templates/oauth-credentials-secret.yaml

```diff
@@ -21,8 +21,10 @@ type: Opaque
 data:
   {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.clientIdKey }}: {{ "client_id" | b64enc | quote }}
   {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.clientSecretKey }}: {{ "client_secret" | b64enc | quote }}
-  {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.oauthUrlKey }}: {{ printf "http://compass-external-services-mock.%s.svc.cluster.local:%s" .Release.Namespace (.Values.service.port | toString) | b64enc | quote }}
+  {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.oauthUrlKey }}: {{ printf "https://compass-external-services-mock-sap-mtls.kyma.local:%s" (.Values.service.certPort | toString) | b64enc | quote }}
```

Comment:
maybe you can use the same thing from https://github.com/kyma-incubator/compass/pull/2198 - `{{ printf "https://%s.%s:%s" .Values.global.externalServicesMock.certSecuredHost .Values.global.ingress.domainName (.Values.service.certPort | toString) | b64enc | quote }}`